cpp/oneapi/dal/backend/primitives/reduction/reduction.hpp

```diff
@@ -31,7 +31,8 @@ sycl::event reduce_by_rows_impl(sycl::queue& q,
                                 ndview<Float, 1>& output,
                                 const BinaryOp& binary,
                                 const UnaryOp& unary,
-                                const event_vector& deps);
+                                const event_vector& deps,
+                                bool override = true);
```

Comment:
Without a description it is not so obvious what this `override` means.

Reply:
Have changed naming. And comments will explain what's going on with this argument anyway.